qp/pdf.py

```diff
@@ -2,6 +2,8 @@
 import scipy.interpolate as spi
 import matplotlib.pyplot as plt
 
+import utils
```

Comment:
I think this needs to become `import qp.utils as utils`. Better would be to add `from utils import *` to `qp/__init__.py` and then do `import qp` followed by `qp.calckl` further down. The point is that `utils` can only be imported like you have done if you are in the `qp` top level folder.